### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by benodonohue**
Hopefully this is right this time..

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -8,22 +8,39 @@ An alternate template for [spike](https://github.com/static-dev/spike) that uses
...
+- `sprout add spike-tpl-roots https://github.com/static-dev/spike-tpl-roots.git`
+- `cd <projectpath>`
+- `sprout new spike-tpl-roots <projectname>`
+
+##### Options
+
+	? What is the name of your project? 
+	? Describe your project 
+	? What is your github username? 
+	? Would you like a production config file? (Y/n)
```
Comment (jescalan) [inline readme.md]: So I still don't see the need to repeat this exact same thing twice. Perhaps we can just put it there once closer to the top?

**Comment by benodonohue** [comment]
Yeah that sounds good.

**Comment by benodonohue** [comment]
I will make the changes when I get home tonight..

**Comment by jescalan** [comment]
Perfect! Thanks for continually putting in the work to make this happen 🎉

**Comment by benodonohue** [comment]
More than happy to contribute mate, roots rocks 🤘Spike is a little
daunting for me at the moment, I just can't get my head around the whole
webpack, cssnext etc thing.

**Pull request merged**